src/cars.json

```diff
@@ -0,0 +1,92 @@
+[
+    {
+        "id": 1,
+        "model": "Mahindra",
+        "description": "4x4 Suv from india short description of the car",
+        "image": "https://i.postimg.cc/YCGv7PmX/Mahindra-Scorpio-N-300620221053-removebg-preview.png"
+    },
```

Comment:
Please kindly add "color", "released_date" and "price" and also implement them in the UI so that you don't need to fix them in the future. Even identical data will be useful.